### ISSUE TITLE
Addition of Greek alphabets

### ISSUE THREAD

**Issue by prakayastha** [ipython/xkcd-font#32]
Hey, thanks a lot for this useful tool. 
However, considering how many scientific people must be using this, it would be nice if greek alphabets are added as well. 

Thanks

**Comment by jbfaden** [comment]
I agree that this should be done.  I've asked a font-loving friend to add a number of glyphs for this, but it's a favor and I don't know how many or when they will be done.  See https://sourceforge.net/p/autoplot/bugs/2196/

**Comment by Tim-the-gitter** [comment]
No experience here, but maybe [this](https://imgs.xkcd.com/comics/greek_letters.png) or [this](https://imgs.xkcd.com/comics/mathematical_symbol_fight.png) may help.

**Comment by Tim-the-gitter** [comment]
> No experience here, but maybe [this](https://imgs.xkcd.com/comics/greek_letters.png) or [this](https://imgs.xkcd.com/comics/mathematical_symbol_fight.png) may help.

BTW: [actual](https://xkcd.com/2586/) [comics](https://xkcd.com/2343/)

**Comment by pelson** [comment]
Excellent suggestion, and very helpful pointers to potential sources of information. Please see #65 for the addition of the full alphabet (with a few creative extensions of the XKCD comics).

**Comment by pelson** [comment]
Note that I have no idea if the font is actually legible... here is an example, for which I can already see that some of the kerning is off, but first and foremost, I would be interested to know if the font even works here...


[<img width="963" height="194" alt="Image" src="https://github.com/user-attachments/assets/d95c2305-a5b7-43b4-abdf-59fbcc9421e4" />](https://ipython.org/xkcd-font/preview.html?text=%F0%9F%87%AC%F0%9F%87%B7%20%CE%A4%CE%B7%20%CF%83%CF%84%CE%B9%CE%B3%CE%BC%CE%AE%20%CF%80%CE%BF%CF%85%20%CF%83%CF%85%CE%BD%CE%B5%CE%B9%CE%B4%CE%B7%CF%84%CE%BF%CF%80%CE%BF%CE%AF%CE%B7%CF%83%CE%B1%20%CF%8C%CF%84%CE%B9%20%CE%B7%20%CF%87%CE%B5%CE%B9%CF%81%CF%8C%CE%B3%CF%81%CE%B1%CF%86%CE%B7%20%CE%B3%CF%81%CE%B1%CE%BC%CE%BC%CE%B1%CF%84%CE%BF%CF%83%CE%B5%CE%B9%CF%81%CE%AC%20%CF%84%CE%BF%CF%85%20XKCD%20%CE%B8%CE%B1%0A%CE%BC%CF%80%CE%BF%CF%81%CE%BF%CF%8D%CF%83%CE%B5%20%CE%BD%CE%B1%20%CF%87%CF%81%CE%B7%CF%83%CE%B9%CE%BC%CE%BF%CF%80%CE%BF%CE%B9%CE%B7%CE%B8%CE%B5%CE%AF%20%CE%B3%CE%B9%CE%B1%20%CF%84%CE%B7%20%CE%B3%CE%BB%CF%8E%CF%83%CF%83%CE%B1%20%CE%BC%CE%BF%CF%85%E2%80%A6%0A%CE%9E%CE%B5%CF%87%CF%89%CF%81%CE%B9%CF%83%CF%84%CE%AE%20%CF%87%CE%B1%CF%81%CE%AC%3A%20%CE%AC%CE%BB%CF%86%CE%B1%20%CE%AD%CF%89%CF%82%20%CF%89%CE%BC%CE%AD%CE%B3%CE%B1%2C%20%CE%AE%CE%BB%CE%B9%CE%BF%CF%82%20%CF%86%CF%89%CF%84%CE%AF%CE%B6%CE%B5%CE%B9%20%CE%BA%CE%B1%CF%81%CE%B4%CE%B9%CE%AD%CF%82%2C%20%CE%B3%CE%BD%CF%8E%CF%83%CE%B7%20%CE%BA%CE%B1%CE%B9%20%CF%86%CF%89%CF%82%20%CE%BF%CE%B4%CE%B7%CE%B3%CE%BF%CF%8D%CE%BD%0A%CE%BC%CF%80%CF%81%CE%BF%CF%83%CF%84%CE%AC%2C%20%CE%BB%CF%8C%CE%B3%CE%BF%CF%82%20%CF%83%CE%BF%CF%86%CF%8C%CF%82.%0A%0A)
